File: esbmc.py/Testes02/validador_titulo/validador_comprimento_titulo.py
MAX_LEN_TITLE: int = 21

def validate_length(len_title: int) -> bool:
    assert 1 <= len_title <= 20, "Tamanho ou caracteres inválidos."
    return True

def main() -> None:
    len_title: int = int(input()) 
    assert 1 <= len_title <= 20     

    validate_length(len_title)

main()

File: esbmc.py/Testes02/validador_titulo/validador_comprimento_titulo.cpp
#define MAX_LEN_TITLE 21

using namespace std;

bool validate_length(int len) {
    __ESBMC_assert(len >= 1 && len <= 20, "Tamanho ou caracteres inválidos.");
    return true;
}

int main() {
    int len;
    __ESBMC_assert(len >= 1 && len <= 20, "Tamanho ou caracteres inválidos.");


    validate_length(len);

    return 0;
}
